webapp/packages/plugin-data-viewer/src/TableViewer/TableViewer.module.css

```diff
@@ -51,6 +51,16 @@
   flex-direction: column;
   overflow: hidden;
 }
+.paneContent.grid:focus-visible::after {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  outline-offset: -1px;
+  outline: -webkit-focus-ring-color auto 1px;
```

Comment:
this one is not working for Firefox

sources:
https://stackoverflow.com/questions/7538771/what-is-webkit-focus-ring-color
https://css-tricks.com/copy-the-browsers-native-focus-styles/

Reply:
fixed